Controller/Adminhtml/Exportcategory/Export.php

```diff
@@ -1,45 +1,50 @@
-* Navin Bhudiya
-* Copyright (C) 2016 Navin Bhudiya <navindbhudiya@gmail.com>
-*
-* NOTICE OF LICENSE
-*
-* This program is free software: you can redistribute it and/or modify
-* it under the terms of the GNU General Public License as published by
-* the Free Software Foundation, either version 3 of the License, or
-* (at your option) any later version.
-*
-* This program is distributed in the hope that it will be useful,
-* but WITHOUT ANY WARRANTY; without even the implied warranty of
-* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-* GNU General Public License for more details.
-*
-* You should have received a copy of the GNU General Public License
-* along with this program. If not, see http://opensource.org/licenses/gpl-3.0.html.
-*
-* @category Navin
-* @package Navin_ImportExportCategory
-* @copyright Copyright (c) 2016 Mage Delight (http://www.navinbhudiya.com/)
-* @license http://opensource.org/licenses/gpl-3.0.html GNU General Public License,version 3 (GPL-3.0)
-* @author Navin Bhudiya <navindbhudiya@gmail.com>
-*/
+ * Navin Bhudiya
+ * Copyright (C) 2016 Navin Bhudiya <navindbhudiya@gmail.com>
+ *
+ * NOTICE OF LICENSE
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see http://opensource.org/licenses/gpl-3.0.html.
+ *
+ * @category Navin
+ * @package Navin_ImportExportCategory
+ * @copyright Copyright (c) 2016 Mage Delight (http://www.navinbhudiya.com/)
+ * @license http://opensource.org/licenses/gpl-3.0.html GNU General Public License,version 3 (GPL-3.0)
+ * @author Navin Bhudiya <navindbhudiya@gmail.com>
+ */
+
 namespace Navin\ImportExportCategory\Controller\Adminhtml\Exportcategory;
 
+use Magento\Framework\Controller\Result\Raw;
+use Magento\Backend\Model\View\Result\ForwardFactory;
+use Magento\Backend\App\Action\Context;
+
 class Export extends \Magento\Backend\App\Action
 {
     /**
      * Redirect result factory
      *
-     * @var \Magento\Backend\Model\View\Result\ForwardFactory
+     * @var ForwardFactory
      */
     protected $_resultForwardFactory;
 
     /**
      * constructor
      *
-     * @param \Magento\Backend\Model\View\Result\ForwardFactory $resultForwardFactory
-     * @param \Magento\Backend\App\Action\Context $context
+     * @param ForwardFactory $resultForwardFactory
+     * @param Context $context
      */
     public function __construct(
         \Magento\Backend\Model\View\Result\ForwardFactory $resultForwardFactory,
```

Comment:
@NathanDayMF let's use consitent `use` statements for all of it.

So @callum-atwal include all classes